packages/nimble-components/src/table/template.ts

```diff
@@ -37,8 +37,17 @@ export const template = html<Table>`
                                 record-id="${(x, c) => c.parent.tableData[x.index]?.id}"
                                 :dataRecord="${(x, c) => c.parent.tableData[x.index]?.record}"
                                 :columns="${(_, c) => c.parent.columns}"
-                                style="height: ${x => x.size}px;"
```

Comment:
Why are we able to remove this in-line `style`?

Reply:
This was not intentional. I think it was a merge mistake on my part. I'll add this back.